L1Trigger/L1TTwinMux/src/IOPrinter.cc

```diff
@@ -99,15 +92,13 @@ cout<<"======================================================"<<endl;
 }
 
 
-void IOPrinter::run(L1MuDTChambPhContainer* inphiDigis,const L1MuDTChambPhContainer & outphiDigis,RPCDigiCollection* rpcDigis,const edm::EventSetup& c) {
+void IOPrinter::run(L1MuDTChambPhContainer const* inphiDigis,const L1MuDTChambPhContainer & outphiDigis,RPCDigiCollection const* rpcDigis,const edm::EventSetup& c) {
 
   cout<<"======================================================"<<endl;
   int bx=0, wheel=0, sector=0, station=1;
-  L1MuDTChambPhDigi const* dtts1=nullptr;
-  L1MuDTChambPhDigi const* dtts2=nullptr;
-  L1MuTMChambPhContainer * inphiDigis_tm=nullptr;
+  L1MuTMChambPhContainer  inphiDigis_tm;
   const std::vector<L1MuDTChambPhDigi> * vInCon=inphiDigis->getContainer();
-  inphiDigis_tm->setContainer(*vInCon);
```

Comment:
This would actually have caused address 0 to be written on (since linke 108 the value is set to nullptr) which would cause a segmentation violation.